File: test_data/code/python/1413-minimum-value-to-get-positive-step-by-step-sum.py
class Solution:
    def minStartValue(self, nums: List[int]) -> int:
        min_start_value = 1
        total = 0
        for num in nums:
            total += num
            if total < 1:
                min_start_value = max(min_start_value, 1 - total)
        return min_start_value

File: test_data/code/java/1413-minimum-value-to-get-positive-step-by-step-sum.java
class Solution {
    public int minStartValue(int[] nums) {
        int sum = 0;
        int minStartValue = 0;

        for (int num : nums) {
            sum += num;
            minStartValue = Math.max(minStartValue, 1 - sum);
        }

        return Math.max(minStartValue, 1);
    }
}